Handle react >= 17

https://github.com/elrumordelaluz/coloreact/blob/283367feb27c8792fb5945b93db0b411d29de403/package.json#L46-L48

Because we get npm errors 🤔
https://github.blog/2021-02-02-npm-7-is-now-generally-available/#peer-dependencies

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my-project
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"~17.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.6.1 || ^16.0.0" from coloreact@0.3.1
npm ERR! node_modules/coloreact
npm ERR!   coloreact@"~0.3.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

See:
 - React 16 to 17: https://reactjs.org/blog/2020/10/20/react-v17.html
 - React 17 to 18: https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html

--- a/package.json
+++ b/package.json
@@ -44,6 +44,6 @@
     "tinycolor2": "^1.3.0"
   },
   "peerDependencies": {
-    "react": "^15.6.1 || ^16.0.0"
+    "react": ">= 15.6.1"
   }
 }

